Dart/src/com/jetbrains/lang/dart/ide/completion/DartServerCompletionContributor.java

```diff
@@ -347,4 +349,25 @@ else if (elementKind.equals(ElementKind.TOP_LEVEL_VARIABLE)) {
       return null;
     }
   }
+
+  @Override
+  public void beforeCompletion(@NotNull final CompletionInitializationContext context) {
```

Comment:
I just invoked the formatter (which maybe is configured to re-arrange?)  I thought this was the canonical thing to do but maybe I don't have the right settings?